rllib/core/rl_module/rl_module.py

```diff
@@ -374,9 +374,17 @@ class RLModule(abc.ABC):
     """
 
     framework: str = None
+    is_learner_module: bool = None
 
     def __init__(self, config: RLModuleConfig):
         self.config = config
+        # By default, each module is a learner module and contains all building blocks,
```

Comment:
nit: Let's use a fixed constant here:

In core/rl_module/__init__.py, add:
```
INFERENCE_ONLY = "_inference_only"
```

Note the underscore to make it harder for users to accidentally override this in their own config dicts.

Reply:
Love it